pod/perlsecpolicy.pod

```diff
@@ -0,0 +1,497 @@
+=encoding utf-8
+
+=for stopwords
+CVE perlsecpolicy SV perl Perl SDBM HackerOne Mitre
+
+=head1 NAME
+
+perlsecpolicy - how we handle security issues
```

Comment:
This abstract and the abstract listed for it in perl.pod should probably match (don't know if that's something we care about?)

Reply:
I can certainly make that change, but checking a handful of pod files listed before and after this one in perl.pod shows that many have similar differences between the two descriptions.

Examples: perlfork, perlnumber, perlthrtut, perlmod, perlmodlib.

Reply:
I switched the abstract to "Perl security report handling policy" in both files in the updated commits.